server/coordinator/scheduler/nodepicker/hash/consistent_uniform.go

```diff
@@ -86,14 +90,14 @@ type virtualNode uint64
 // consistent as possible while the members has some tiny changes.
 type ConsistentUniformHash struct {
 	config        Config
-	minLoad       float64
-	maxLoad       float64
+	minLoad       int
+	maxLoad       int
 	numPartitions uint32
 	// Member name => Member
 	members map[string]Member
-	// Member name => Member's load
-	memLoads map[string]float64
-	// partition id => index of the virtualNode in the sortedRing
+	// Member name => Partition ID
```

Comment:
This line of comments does not seem to correspond to the code. Why the following code is `map[int]struct`?